code/modules/surgery/organs/stomach/external/wings.dm

```diff
@@ -7,10 +7,9 @@
 	preference = "wings"
 
 /obj/item/organ/external/wings/can_draw_on_bodypart(mob/living/carbon/human/human)
-	if(!(human.wear_suit?.flags_inv & HIDEJUMPSUIT))
-		if( !(human.wear_suit?.species_exception && is_type_in_list(src, human.wear_suit.species_exception) ) )
-			return TRUE
-	return FALSE
+	if(human.wear_suit?.flags_inv & HIDEJUMPSUIT && !(human.wear_suit.species_exception && is_type_in_list(src, human.wear_suit.species_exception)))
```

Comment:
Nono--split this into multiple ifs, but early return.

As in...


```dm
if (!human.wear_suit?.flags_inv & HIDEJUMPSUIT)
    return TRUE

// etc
```

Reply:
Oh, I misunderstood
One moment...